packages/runner/src/pages/CustomFunctionsRunner/setup.ts

```diff
@@ -34,11 +34,16 @@ export default () => {
       case 'metadata': {
         const initialPayload = payload as ICustomFunctionsIframeRunnerOnLoadPayload;
 
-        await addScriptTag(
-          initialPayload.pythonConfig
-            ? SCRIPT_URLS.CUSTOM_FUNCTIONS_RUNNER_WITH_JUPYTER_SUPPORT
-            : SCRIPT_URLS.CUSTOM_FUNCTIONS_RUNNER_DEFAULT,
-        ).then(() => {
+        let scriptUrl = '';
+        if (initialPayload.pythonConfig) {
```

Comment:
Nit:  I think this would be better as a helper function.  And that way you can just have `const scriptUrl = getScriptUrl()`, with a helper function that always returns rather than setting a variable.

Reply:
Added `getScriptUrl()`. Thanks for the suggestion!